spec/index.bs

```diff
@@ -7014,214 +7002,214 @@ The "Filter" column specifies whether textures of this format can be sampled in
     </thead>
```

Comment:
nit on unchanged line above this: "Component Types" should be "Component Type" I think